lib/Bio/EnsEMBL/DataCheck/index.json

```
@@ -403,6 +403,16 @@
       "name" : "CheckHomologyMLSS",
       "package_name" : "Bio::EnsEMBL::DataCheck::Checks::CheckHomologyMLSS"
    },
+   "CheckIDRangeMLSS" : {
+      "datacheck_type" : "advisory",
+      "description" : "All relevant IDs are within the offset range for their MLSS ID",
+      "groups" : [
+         "compara",
+         "compara_pairwise_alignments"
```

Following my previous comment:
```suggestion
         "compara_pairwise_alignments",
         "compara_genome_alignments"
```